### ISSUE TITLE
Selecting mat-option which has no value set fires valueChange and selectionChange events twice.

### ISSUE THREAD

**Issue by danfan1** [angular/material2#12267]
#### What is the expected behavior?
When a mat-option with no value set, the event valueChange and selectionChange events are fired once.

#### What is the current behavior?
Those two events are fired twice.

#### What are the steps to reproduce?
https://angular-e2cc3n.stackblitz.io
Select the first option, which has no value set, and look at the console log, you will see the two events are fired twice.

#### What is the use-case or motivation for changing an existing behavior?
In some use cases, some XHRs are triggered by responding to these events. When the event is fired twice, the XHRs is triggered twice, which may have unexpected effect.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Material 6.3.3

**Comment by crisbeto** [comment]
Duplicate of https://github.com/angular/material2/issues/10675.

**Comment by danfan1** [comment]
This is not exactly the same as #10675.
#10675 is about when the option is selected the second time, change event is fired, but shouldn't.
This bug is about when the option is selected the first time, change event is fired twice, but should be fired only once.

**Comment by crisbeto** [comment]
I think the fix should cover both cases, otherwise we can reopen this one.
